flang/include/flang/Optimizer/Analysis/AliasAnalysis.h

```diff
@@ -157,6 +157,17 @@ struct AliasAnalysis {
     bool isData() const;
     bool isBoxData() const;
 
+    /// Check conditions related to dummy argument aliasing.
+    ///
+    /// For all uses, a result of false can prevent MayAlias from being
+    /// reported, so the list of cases where false is returned is conservative.
+    /// @{
+    bool aliasesLikeDummyArg() const;
+    bool aliasesLikePtrDummyArg() const;
+    bool canBeActualArg() const;
+    bool canBeActualArgWithPtr(const mlir::Value *val) const;
```

Comment:
Please add some short explanation, it is not very clear what is meant looking at the names it could be read as implying that they can be passed as actual argument in call inside the current function, not that they can be actual argument of the current function.

Reply:
Done.